examples/research_projects/jax-projects/README.md

```diff
@@ -116,7 +118,232 @@ Additionally, the organizers, other participants, or anybody in the community re
+While we recommend that all technical aspects of work can be discussed directly on the [forum](https://discuss.huggingface.co/c/flax-jax-projects/22) under your project thread, 
+it can be very helpful to have a more direct way of communicating, *e.g.* in a channel or 
+For this we have created a discord that you can access [here](https://discord.com/channels/858019234139602994/858019234139602997). 
+This discord will not be managed by anybody and is just there so that you can communicate more effectively with your team members. 
+Feel free to create a new channel for you and your team where you can discuss everything. If you and your team have already set up other ways of communicating, it is absolutely not required to make use of the discord. However, we do recommend each team to set up some kind 
+of channel or group for quick discussions.
+
+### Project definition
+
+In the very beginning, you should make sure your project is well-defined and that 
+everybody in the team understands the goal of the project and the work that needs to be 
+done in order to achieve the goal. A well-defined project:
+
+- has defined the task on which the model will be trained
+- has defined the model that will be trained
+- has defined the datasets that will be used for training
+- has defined the type of training scripts that need to be written
+- has defined the desired outcome of the project
+- has defined the work flows
+
+By "has defined" it'ss not meant that the corresponding code is already written and ready 
+to be used, but that everybody in team is on the same page on what type of model, data and training script should be used.
+
+To give an example, a well-defined project would be the following:
+
+- task: summarization
+- model: [t5-small](https://huggingface.co/t5-small)
+- dataset: [CNN/Daily mail](https://huggingface.co/datasets/cnn_dailymail)
+- training script: [run_summarization_flax.py](https://github.com/huggingface/transformers/blob/master/examples/flax/summarization/run_summarization_flax.py)
+- outcome: t5 model that can summarize news
+- work flow: adapt `run_summarization_flax.py` to work with `t5-small`.
+
+This example is a very easy and not the most interesting project since a `t5-small`
+summarization model exists already for CNN/Daily mail and pretty much no code has to be 
+written. 
+A well-defined project does not need to have the dataset be part of 
+the `datasets` library and the training script already be pre-written, however it should 
+be clear how the desired dataset can be accessed and how the training script can be 
+written. 
+
+It is also important to have a clear plan regarding the work flow. Usually, the 
+data processing is done in a first step. Once the data is in a format that the model can 
+work with, the training script can be written, etc. These steps should be more detailed 
+once the team has a clearly defined project. It can be helpful to set deadlines for each step.
```

Comment:
```suggestion
It is also important to have a clear plan regarding the workflow. Usually, the 
data processing is done in a first step. Once the data is in a format that the model can 
work with, the training script can be written, etc. These steps should be more detailed 
once the team has a clearly defined project. It can be helpful to set deadlines for each step.
```